parts/linux/cloud-init/artifacts/mariner/cse_helpers_mariner.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+echo "Sourcing cse_helpers_distro.sh for Mariner"
+
+aptmarkWALinuxAgent() {
+    echo "No aptmark equivalent for DNF"
+}
```

Comment:
Keep this empty method here so it reminds us the diff between Ubuntu and Mariner, in case we run into package installation issues due to walinuxagent update.

Also rename to dnfmarkWALinuxAgent?